interface/patient_file/pos_checkout.php

```diff
@@ -52,9 +52,7 @@
  * @link    http://www.open-emr.org
  */
```

Comment:
after address issues brought up in this script, it will be good to go (history was also checked)